test/tstInterpDetailsSVD.cpp

```diff
@@ -0,0 +1,134 @@
+/****************************************************************************
+ * Copyright (c) 2023 by the ArborX authors                                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "ArborX_EnableDeviceTypes.hpp"
+#include "ArborX_EnableViewComparison.hpp"
+#include <interpolation/details/ArborX_InterpDetailsSymmetricPseudoInverseSVD.hpp>
+
+#include "BoostTest_CUDA_clang_workarounds.hpp"
+#include <boost/test/unit_test.hpp>
+
+namespace axid = ArborX::Interpolation::Details;
```

Comment:
It stands for 'ArborX Interpolation Details'. It is simply a namespace shortcut. But because it is not used in a lot of places, I will use the full namespace name.